src/main/java/smartthings/cassandra/CassandraLock.java

```diff
@@ -31,22 +33,35 @@ public CassandraLock(CassandraConnection cassandraConnection, int ttl) {
 
 		setupTables();
 
-		insertLock = session.prepare("INSERT INTO databasechangelock(id, lockedby) VALUES (:lockId, :owner) IF NOT EXISTS USING TTL :ttl");
-		insertLock.setConsistencyLevel(ConsistencyLevel.QUORUM);
-
-		deleteLock = session.prepare("DELETE FROM databasechangelock WHERE id = :lockId IF lockedby = :owner");
-		deleteLock.setConsistencyLevel(ConsistencyLevel.QUORUM);
-
-		selectLock = session.prepare("SELECT lockedby, TTL(lockedby) AS ttl FROM databasechangelock WHERE id = :lockId");
-		selectLock.setConsistencyLevel(ConsistencyLevel.SERIAL);
-
-		updateLock = session.prepare("UPDATE databasechangelock USING TTL :ttl SET lockedby = :owner WHERE id = :lockId IF lockedby = :owner");
-		deleteLock.setConsistencyLevel(ConsistencyLevel.QUORUM);
+		insertLock = session.prepare(
+			SimpleStatement.newInstance("INSERT INTO databasechangelock(id, lockedby) VALUES (:lockId, :owner) IF NOT EXISTS USING TTL :ttl")
+				.setConsistencyLevel(DefaultConsistencyLevel.QUORUM)
+		);
+
+		deleteLock = session.prepare(
+			SimpleStatement.newInstance("DELETE FROM databasechangelock WHERE id = :lockId IF lockedby = :owner")
+				.setConsistencyLevel(DefaultConsistencyLevel.QUORUM)
+		);
+
+		selectLock = session.prepare(
+			SimpleStatement.newInstance("SELECT lockedby, TTL(lockedby) AS ttl FROM databasechangelock WHERE id = :lockId")
+				.setConsistencyLevel(DefaultConsistencyLevel.SERIAL)
+		);
+
+		updateLock = session.prepare(
+			SimpleStatement.newInstance("UPDATE databasechangelock USING TTL :ttl SET lockedby = :owner WHERE id = :lockId IF lockedby = :ifowner")
```

Comment:
This is a bit obnoxious.  All the bind variable - set by name methods appear only to support setting the first occurrence.
See:
https://github.com/datastax/java-driver/blob/4.x/core/src/main/java/com/datastax/oss/driver/api/core/data/SettableByName.java

Best solution I've determined so far is just make each bind name unique.. Couldn't find a method/impl that automagically set them all.  If you leave it the same it results in a RuntimeException essentially saying you have unset binds.